internal/utils.go

```diff
@@ -183,6 +191,26 @@ func getPolicyTemplates(policyConf *types.PolicyConfig) ([]map[string]map[string
 	return policyTemplates, nil
 }
 
+// checkPolicyTypeManifest determine if the manifest is policy-type manifest.
+func checkPolicyTypeManifest(manifest map[string]interface{}) bool {
+	apiVersion, foundAPIVersion, isStr := unstructured.NestedString(manifest, "apiVersion")
+	if !foundAPIVersion || isStr != nil {
```

Comment:
I think you can remove this if statement since in the error case, `apiVersion` is an empty string so the `HasPrefix` below would still be false.

Reply:
I check these in case apiVersion or kind are objects rather than string then HasPrefix will throw some error.